mattermost-plugin/webapp/package.json

```diff
@@ -68,17 +68,15 @@
     "sass-loader": "11.1.1",
     "style-loader": "2.0.0",
     "ts-loader": "9.2.3",
+    "typescript": "4.2.4",
     "webpack": "5.37.1",
     "webpack-cli": "4.7.0"
   },
   "dependencies": {
     "core-js": "3.12.1",
     "mattermost-redux": "5.33.1",
-    "react": "17.0.2",
     "react-intl": "^5.13.5",
-    "react-redux": "7.2.4",
-    "redux": "4.1.0",
-    "typescript": "4.2.4"
+    "react-router-dom": "5.2.0"
```

Comment:
This line fix the problem, the other lines just clean this up to not include the not-needed dependencies (they are used from the mattermost-webapp).